include/dem/explicit_euler_integrator.h

```diff
@@ -49,18 +49,29 @@ class ExplicitEulerIntegrator : public Integrator<dim>
   {}
 
   /**
-   * Carries out the integration of the motion of all particles by using
-   * the acceleration with the explicit Euler method
+   * Carries out the prediction (pre_force) integration calculations.
+   *
+   * @param particle_handler The particle handler whose particle motion we wish
+   * to integrate
+   * @param time_step The value of the time step used for the integration
+   */
+  virtual void
+  integrate_pre_force(Particles::ParticleHandler<dim> &particle_handler,
+                      double                           time_step) override;
```

Comment:
Shouldnt the Body force be passed as an argument here also for generality? it's kinda weird to have different arguments for both. I know Verlet does not use it but still.